android/java/org/chromium/chrome/browser/settings/BraveWalletNetworksPreference.java

```diff
@@ -0,0 +1,132 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.chromium.chrome.browser.settings;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.widget.TextView;
+
+import androidx.preference.Preference;
+import androidx.preference.PreferenceViewHolder;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import org.chromium.brave_wallet.mojom.BraveWalletConstants;
```

Comment:
it seems 
`import org.chromium.brave_wallet.mojom.BraveWalletConstants;` 
`import java.util.ArrayList;` 
`import java.util.List;`
are not used